### PR TITLE
fix session lang is null

### PR DESCRIPTION
**Pull request by futabooo**
## Issue
- #117

## Overview (Required)
- fix Resources$NotFoundException when session lang is null

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by konifar** [comment]
👀

File: app/src/main/java/io/github/droidkaigi/confsched2017/util/StringUtil.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package io.github.droidkaigi.confsched2017.util;
+
+public class StringUtil {
+
+    public static boolean isHalfWidthCharacters(String s) {
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/util/StringUtil.java]: I think it's too much to check the characters 🤔 
If `lang` is null, it should not be bind `lang` to the view and it's better to hide it like this.
https://github.com/DroidKaigi/conference-app-2017/pull/115#discussion_r99573201

What do you think?

**Comment by konifar** [comment]
Thanks for contribution! Commented 😃

File: app/src/main/java/io/github/droidkaigi/confsched2017/util/StringUtil.java (hunk 1 above)
Comment (futabooo) [inline app/src/main/java/io/github/droidkaigi/confsched2017/util/StringUtil.java]: Thank you for review.
I hustle too much 😆 
you are right!

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/util/StringUtil.java]: Anyway, your code is interesting!! 👍 
I learned the logic from your code 📝

File: app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionDetailViewModel.java [hunk 2]
```diff
@@ -83,6 +84,12 @@ private void setSession(@NonNull Session session) {
 
         if (session.lang != null) {
             this.languageResId = decideLanguageResId(session.lang.toUpperCase());
+        } else {
+            boolean englishTitle = StringUtil.isHalfWidthCharacters(session.title);
+            if (englishTitle) {
+                this.languageResId = decideLanguageResId(Session.LANG_EN_ID);
+            }
+            this.languageResId = decideLanguageResId(Session.LANG_JA_ID);
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionDetailViewModel.java]: Does this need `else` ?

**Comment by konifar** [comment]
Sorry, I'll close this PR 🙇 
Thank you for your contribution!!
I would be happy if you will send another PR 🙏

File: app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionDetailViewModel.java (hunk 2 above)
Comment (futabooo) [inline app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionDetailViewModel.java]: oh,, right

**Comment by futabooo** [comment]
yes, i will 💪

**Pull request closed without merging**